source/frontend/src/features/documents/ComposedDocument.ts

```diff
@@ -50,6 +50,22 @@ export class DocumentRow {
     return row;
   }
 
+  public static fromApiDocument(document: Api_Document): DocumentRow {
+    const row: DocumentRow = new DocumentRow();
+    row.id = document?.id;
+    row.documentType = document?.documentType;
+    row.mayanDocumentId = document?.mayanDocumentId;
+    row.statusTypeCode = document?.statusTypeCode;
+    row.fileName = document?.fileName;
+    row.appCreateTimestamp = document?.appCreateTimestamp;
+    row.appCreateUserid = document?.appCreateUserid;
+
+    row.relationshipId = undefined;
+    row.relationshipType = undefined;
+    row.parentId = undefined;
```

Comment:
this looks like the doc row might be a Document Relationship representation rather than just a document. Do you know if passing a document relationship works better?

Reply:
yes, this was just added for compatibility purposes. In some cases a row is built from a Document Relationship and sometimes from a document. Previously we were getting away with this because of the loose type checking.

Reply:
I have some time now since my other stories are done, can probably go through and try and make the usage consistent.